docs/developers/architecture/napari_models.md

```diff
@@ -1,4 +1,6 @@
-# Napari models and events
+(napari-model-event)=
+
+# napari models and events
```

Comment:
Ugh, can't make suggestion, but can we update this to use `Python` instead of `python`? It's how we do it on other pages, plus it's the preferred styling.

Reply:
Done!

Reply:
There's a few more in that section, if you scroll down a bit...
Sorry!

Reply:
No worries, done!